README.md

```diff
@@ -37,6 +37,10 @@ brew tap gitguardian/tap
 brew install src-fingerprint
 ```
 
+### Linux packages
+
+Deb and RPM packages are available on [Cloudsmith](https://cloudsmith.io/~gitguardian/repos/src-fingerprint/packages/).
```

Comment:
I think we should add the commands to install the package using cloudsmith.

Reply:
The instructions are quite long (especially on the RPM side, with all the variants), so I added direct links to Cloudsmith instructions. Is it good enough for you?